src/qibo/backends/__init__.py

```diff
@@ -33,6 +33,14 @@ def construct_backend(backend, platform=None):
     elif backend == "qibolab":  # pragma: no cover
         from qibolab.backends import QibolabBackend  # pylint: disable=E0401
 
+    elif backend == "qibotn":  # pragma: no cover
+        from qibotn.backends import CuTensorNet, QuimbBackend  # pylint: disable=E0401
+
+        if platform == "cutensornet":  # pragma: no cover
+            return CuTensorNet(runcard)
```

Comment:
`runcard` is currently undefined, but after #1157 you will be able to retrieve it from arbitrary keyword arguments.
```suggestion
            return CuTensorNet(kwargs["runcard"])
```

Reply:
Thanks for the rebase, I have made the changes